io_scene_nif/nif_common.py

```diff
@@ -50,21 +49,7 @@ class NifCommon:
     that are commonly used in both import and export.
     """
 
-    VERTEX_RESOLUTION = 1000
-    NORMAL_RESOLUTION = 100
-
-    EXTRA_SHADER_TEXTURES = [
-        "EnvironmentMapIndex",
-        "NormalMapIndex",
-        "SpecularIntensityIndex",
-        "EnvironmentIntensityIndex",
-        "LightCubeMapIndex",
-        "ShadowTextureIndex"]
-    """Names (ordered by default index) of shader texture slots for
-    Sid Meier's Railroads and similar games.
-    """
-
-    HAVOK_SCALE = 6.996
+    SELECTED_OBJECTS = []
```

Comment:
The other thing is that the set of selected geometries updates through the import process, so we need to get the initial list at the start, otherwise could just use `bpy.data.object.selectedobject[:]`